core/store/src/columns.rs

```diff
@@ -342,6 +342,22 @@ pub enum DBCol {
     /// - *Content type*: next block: Vec<BlockHash (CryptoHash)>
     #[cfg(feature = "protocol_feature_spice")]
     AllNextBlockHashes,
+    /// For spice contains execution results endorsements.
+    /// - *Rows*: ChunkProductionKeyWithAccount ([near_primitives::stateless_validation::ChunkProductionKey] || AccountId)
+    /// - *Content type*: ([near_primitives::stateless_validation::chunk_endorsement::SpiceEndorsementSignedInner], [near_crypto::Signature])
```

Comment:
I suggest creating a dedicated type for that instead of using raw tuple, something like `SpiceEndorsementWithSignature`